monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+# Triptych: Un Nouvel Algorithme Protégeant les Données des Utilisateurs de Monero
+
+**“Monero obfusque l'émetteur, le destinataire, et le montant des transactions. Triptych est une amélioration pour Monero qui rend son utilisation plus rapide, plus légère et plus privée.” - 1e septembre, 2020**
+
+Monero est conçu pour obfusquer les valeurs (émetteur, destinataire et montants) de chaque transaction avec des techniques et des technologies évolutives. Il s’agit de présenter ici Triptych, une technologie innovante permettant l’optimisation de la fonction d'obfuscation de l’émetteur, celui de l’usage de la chaine de blocs et une rapidité d’exécution supérieure.
+
+### _Les Signatures de Cercle_
+
+Pour assurer la confidentialité des émetteurs, Monero fait plus que garder le secret des adresses. L’ensemble de la base des données de Monero existe sur sa chaine de blocs sous forme de transactions sortantes dénommées TXO. À chaque TXO unique est assigné une identité unique (ID), appelée clé publique (également nommée "adresse furtive"). Par simple analyse de la chaine de blocs de Monero, il est impossible d'établir une corrélation entre les clés publiques des TXO et les adresses des portefeuilles électroniques de Monero. Cela étant, les clés publiques ont également besoin d’une protection. Il serait inacceptable que les flux de Monero parmi les clés publiques soient transparents, considérant que des techniques d'analyses externes pourraient associer ces clés publiques a des portefeuilles, puis à des individus.
```

Comment:
1.
À chaque TXO unique est assigné une identité unique (ID) >>> À chaque TXO est assigné une identité unique (ID)   

Reason: The first 'unique' is not necessary because 'chaque' already contain this meaning

2.
corrélation  >>> relation   
Reason: 'corrélation' correspond to a mutual relation, in both ways. But here they refer to a one way relation so that 'relation' is more appropriate.

3.
associer ces clés publiques a des portefeuilles, puis à des individus.  >>> associer ces clés publiques à des individus.  
Reason: The original did not mention the term wallet ('portefeuille'). I understand that this could slightly clarify the meaning of the text, but I do not think it was really necessary to add it.

Reply:
Agreed on all counts. Integrated.